lib/middleware/request-handler.ts

```diff
@@ -406,11 +412,17 @@ export function elevateRequest (req: Interface.IOurRequest,
         res.write(text + JSON.stringify(data));
     };
 
-    res.sendEnd = (status: string, message: string): void => {
-        var text = util.format('%s%s,%s}',
+    res.sendEnd = (status: string,
```

Comment:
I wouldn't like to have the correlation ids as one of the element of the data array and that they may be intermixed with elements of a different kind (even if that is a rare occurence this case still needs to be handled). It may simplify our code but will complexify the handling of the response.
Extending (optionally) the response with some generic properties is not that complex and will only add an optional parameter to sendEnd.